jujugui/static/gui/src/app/views/utils.js

```diff
@@ -570,7 +570,19 @@ YUI.add('juju-view-utils', function(Y) {
       console.log(endpoints);
       result = {error: 'Specified relation is unavailable.'};
     } else if (matches.length > 1) {
-      result = {error: 'Ambiguous relationship is not allowed.'};
+      // It's only a problem if there's more than one explicit relation.
+      // Otherwise, filter out the implicit relations and just return the
+      // explicit relation.
+      var explicitRelations = matches.filter(
+        rel => rel['interface'] !== 'juju-info');
```

Comment:
Can you check what the providerType name is at this point? I'm pretty sure that's what needs to be checked instead of the interface to match the juju-core implementation.